frontend/appflowy_flutter/packages/flowy_svg/lib/src/flowy_svg.dart

```diff
@@ -23,7 +23,7 @@ class FlowySvg extends StatelessWidget {
     this.size,
     this.color,
     this.blendMode = BlendMode.srcIn,
-    this.opacity = 1.0,
```

Comment:
I wonder there might some svg widgets rely on `opacity=1.0.` default value. So it might cause UI diplay issue

Reply:
Hmm... AFAIK, color values in our themes are copied out of the figma file without any modification, so purposefully changing the opacity should be situational like applying a visual effect according to an user interaction.

We should remove defaulting to 1.0 it because semi-transparent color copied from our mockups doesn't look the same in app. The opposite holds true; if we notice that there is an icon that looks off in app, then it should also look off in the mockups and we should improve the design.